lib/timeline/Core.js

```diff
@@ -705,9 +724,15 @@ Core.prototype._redraw = function() {
   // calculate the widths of the panels
   props.root.width = dom.root.offsetWidth;
   props.background.width = props.root.width - borderRootWidth;
-  props.left.width = dom.leftContainer.clientWidth   || -props.border.left;
+
+  var scrollbarWidth = 0;
+  if (this.options.verticalScroll) {
+    scrollbarWidth = 16;
```

Comment:
Sadly the width of the scrollbar can be different in each browser. You need a hack to get it. Here is a nice explanation: http://stackoverflow.com/a/13382873/722162